include/cetl/pf20/span.hpp

```diff
@@ -329,7 +331,7 @@ class span
     /// If the span has a zero size or not.
     /// @return true if the span size is 0 where "size" is the same as span::extent
     /// for this specialization.
-#if (__cplusplus >= CETL_CPP_STANDARD_17)
+#if (__cplusplus >= 201703L)
```

Comment:
why the number change to 17 #defines?

Reply:
Since the inclusion of cetl.h asserts that this is a value that we expect to see and refuses to use anything like `201704L` etc I've decided to reduce the number of places within this type we need something named "CETL" to compile.